### PR TITLE
card #56722 remove the mentioning of Deriv Europe Limited on binary.com

### PR DESCRIPTION
**Pull request by yauheni-deriv**

### CODE REVIEW

**Review by ali-hosseini-fs — DISMISSED** [review]
LGTM

**Review by matin-binary — APPROVED** [review]

**Review by hojjat-fs — APPROVED** [review]

File: src/templates/static/legal/tac_sections/tac.jsx [hunk 1]
```diff
@@ -3,14 +3,13 @@ import React from 'react';
 const Tac = () => (
     <div>
         <h2 data-anchor='terms-and-conditions'>{it.L('Terms and conditions')}</h2>
-        <p><i>{it.L('Version 52. Last modified 26-11-2021.')}</i></p>
+        <p><i>{it.L('Version 53. Last modified 10-02-2022.')}</i></p>
```
Comment (yauheni-deriv) [inline src/templates/static/legal/tac_sections/tac.jsx:6]: Please consider, the last agreed date is 01-03-2022
it should be changed with PR https://github.com/binary-com/binary-static/pull/7055

File: src/templates/static/legal/tac_sections/tac.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const Tac = () => (
     <div>
         <h2 data-anchor='terms-and-conditions'>{it.L('Terms and conditions')}</h2>
-        <p><i>{it.L('Version 53. Last modified 10-02-2022.')}</i></p>
+        <p><i>{it.L('Version 53. Last modified 01-03-2022.')}</i></p>
```
Comment (yauheni-deriv) [inline src/templates/static/legal/tac_sections/tac.jsx:6]: the date was changed due to requirements of other task and slack discussion.

https://redmine.deriv.cloud/issues/52818#Update_PA_T&Cs_(Binary)
https://deriv-group.slack.com/archives/C02E0CSRBUJ/p1644302935386899

**Review by hojjat-fs — APPROVED** [review]

File: src/templates/static/legal/tac_sections/tac.jsx (hunk 2 above)
Comment (matin-binary) [reply src/templates/static/legal/tac_sections/tac.jsx:6]: You can remove the external links as our codebase is open source!

**Review by matin-binary — APPROVED** [review]

**Pull request merged**